src/cli/crawl-specs.js

```diff
@@ -368,45 +370,21 @@ async function saveResults(crawlInfo, crawlOptions, data, folder) {
             }));
     };
 
-    // Helper function that returns true when the given spec is is the latest
-    // level of that spec in the crawl for the given type of content
-    // ("css" or "idl"). Note the code handles the special case of the CSS2
-    // and CSS22 specs, and assumes that URLs that don't end with a level
-    // number are at level 1 (this does not work for CSS specs whose URLs still
-    // follow the old `css3-` pattern, but we're only interested in comparing
-    // with more recent levels in that case, so it does not matter)
-    const isLatestLevel = (spec, flag) => {
-        const getLevel = spec =>
-            (spec.url.match(/-\d+\/$/) ?
-            parseInt(spec.url.match(/-(\d+)\/$/)[1], 10) :
-            (spec.url.match(/CSS22\/$/i) ? 2 : 1));
-        const shortname = getShortname(spec);
-        const level = getLevel(spec);
-        const candidates = data.filter(s => s.flags[flag] &&
-            (getShortname(s) === shortname) && (getLevel(s) >= level));
-
-        // Note the list of candidates for this shortname includes the spec
-        // itself. It is the latest level if there is no other candidate at
-        // a strictly greater level, and if the spec under consideration is
-        // the first element in the list (for the hopefully rare case where
-        // we have two candidate specs that are at the same level)
-        return !candidates.find(s => getLevel(s) > level) &&
-            (candidates[0] === spec);
-    };
-
     // Save IDL dumps for the latest level of a spec to the idl folder
     await Promise.all(data
-        .filter(spec => spec.flags.idl && spec.idl && spec.idl.idl)
-        .filter(spec => isLatestLevel(spec, 'idl'))
+        .filter(spec => !spec.flags.delta)
+        .filter(spec => spec.idl && spec.idl.idl)
+        .filter(spec => isLatestLevel(spec, data))
         .map(saveIdl));
 
     // Save CSS dumps for the latest level of a spec to the css folder
     await Promise.all(data
-        .filter(spec => spec.flags.css && spec.css && (
+        .filter(spec => !spec.flags.delta)
```

Comment:
`isLatestLevel` is documented as "The function ignores delta specs, which are never considered to be the last level" so could the delta check be dropped here and elsewhere?

Reply:
I'll update the documentation. I didn't mean "ignores" as "does not handle" but rather as "makes sure that delta specs are not considered when looking for a newer level".